.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 2022,
+    "sourceType": "module"
+  },
+  "plugins": [
+    "@typescript-eslint",
+    "eslint-plugin-no-only-tests",
+    "eslint-plugin-import"
+  ],
+  "rules": {
+    "import/extensions": ["error", "always"],
```

Comment:
Should we enable `import/no-extraneous-dependencies` too? Some packages like custom-elements-manifest-tools may be published to npm, so it'll matter.

Reply:
Done